java/build.xml

```diff
@@ -47,5 +48,6 @@
   <target name="dist" depends="jar,test">
     <!-- copy and rename the library -->
     <copy file="build/vm_service_lib.jar" tofile="dist/vm_service_lib-${service.version}.jar"/>
+    <copy file="build/vm_service_lib_src.jar" tofile="dist/vm_service_lib_src-${service.version}.jar"/>
```

Comment:
Perhaps put a dev version suffix in the init section, and then use it here to build up the file name?